eng/build.ps1

```diff
@@ -352,6 +354,10 @@ function TestUsingOptimizedRunner() {
     $env:ROSLYN_TEST_IOPERATION = "true"
   }
 
+  if ($testUsedAssemblies) {
+    $env:ROSLYN_TEST_USEDASSEMBLIES = "true"
```

Comment:
Should we consider adding a new CI leg to run this check? /cc @jaredpar.

Reply:
> Should we consider adding a new CI leg to run this check? 

I think we should do this once the feature is moved to master.

---
In reply to: [369228944](https://github.com/dotnet/roslyn/pull/40987#discussion_r369228944) [](ancestors = 369228944)